numba_cuda/numba/cuda/tests/support.py

```diff
@@ -817,3 +821,28 @@ def check_warnings(self, messages, category=RuntimeWarning):
                     self.assertEqual(w.category, category)
                     found += 1
         self.assertEqual(found, len(messages))
+
+
+class SerialMixin(object):
+    """Mixin to mark test for serial execution."""
+
+    _numba_parallel_test_ = False
+
+
```

Comment:
The `SerialMixin` carries no meaning in the Numba-CUDA testsuite (it only has an effect with the Numba test runner):
```suggestion
```